src/net_processing.h

```diff
@@ -146,6 +176,16 @@ class PeerManager final : public CValidationInterface, public NetEventsInterface
     CTxMemPool& m_mempool;
 
     int64_t m_stale_tip_check_time; //!< Next time to check for stale tip
+
+    /** Protects m_peer_map */
```

Comment:
322e87b6039877ed3457baea93cc87d78f3f4d96
nit: This comment duplicates the code :) Consider to drop it.

Reply:
Normally I'd agree with removing unnecessary commenting, but I think it's useful for every member variable to have a doxygen comment. Happy to go either way if others think it should be dropped.

Reply:
I'm for keeping it, it's better for consistency to have a comment for every one.